### PR TITLE
Add JSONPath to utility libraries

### PR DESCRIPTION
**Pull request by gelraen**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -121,6 +121,7 @@
 - [tanaikech/RangeListApp](https://github.com/tanaikech/RangeListApp) RangeListApp is a GAS library for retrieving, putting and replacing values for Spreadsheet by a range list with a1Notation using Google Apps Script (GAS)
 - [syslogic/google-apps-script](https://github.com/syslogic/google-apps-script) CloudDatastore.gs is a client for Google Cloud Datastore, which runs as a Service Account
 - [Sheetbase](https://sheetbase.net) Build REST API server, websites and apps using Google Apps Script.
+- [JSONPath](https://script.google.com/a/google.com/d/1BtmcVnmTKAxqdiHN9Q5cM6M6E5MUMzVtQjSfav-OIQxNIoACkDdlst53/edit) Extract data from complex data structures with simple expressions. Google Apps Script version of [this](https://github.com/dchester/jsonpath) library.
```
Comment (oshliaer) [inline readme.md]: It is a good tool, but it doesn't have an open publication such as a github or gitlab repository, etc. Please move this line under the heading [Snippets and assorted](https://github.com/oshliaer/google-apps-script-awesome-list#snippets-and-assorted)

Comment (gelraen) [reply readme.md]: Done, PTAL

Comment (oshliaer) [reply readme.md]: Great! Thanks so much!

**Comment by oshliaer** [comment]
Thanks for your support :rocket:

**Pull request merged**